docs/UsersGuide/source/BuildRunSRW.rst

```diff
@@ -601,7 +633,55 @@ For WCOSS_CRAY:
    ``valid_param_vals script``. In addition, various example configuration files can be
    found in the ``regional_workflow/tests/baseline_configs`` directory.
 
+.. _VXConfig:
+
+Configure METplus Verification Suite (Optional)
+--------------------------------------------------
+
+Users who want to use the METplus verification suite to test their forecasts need to load the appropriate modules and add additional information to their ``config.sh`` file. Other users may skip to the :ref:`next section <SetUpPythonEnv>`. 
+
+.. note::
+   METplus is preinstalled on `Level 1 <https://github.com/ufs-community/ufs-srweather-app/wiki/Supported-Platforms-and-Compilers>`__ systems. METplus *installation* is currently not supported for this release of the SRW App, but METplus *use* is supported on systems with a functioning METplus installation. For more information about METplus, see :numref:`Section %s <MetplusComponent>`.
+
+Once installed, METplus users must load the appropriate modules (changing the module location and MET version to correspond to their system's installation):
+
+.. code-block:: console
+   
+   module use -a </path/to/met/modulefiles/>
+   module load met/<10.0.0>
+
+Then, the path to the MET and METplus directories must be added to ``config.sh``:
+
+.. code-block:: console
 
+   METPLUS_PATH="</path/to/METplus/METplus-4.1.0>"
+   MET_INSTALL_DIR="</path/to/met/10.1.0>"
+
+Users who have already staged the METplus verification data (i.e., the :term:`CCPA`, :term:`MRMS`, and :term:`NDAS` data) on their system should set the path to this data and set the corresponding ``RUN_TASK_GET_OBS_*`` parameters to "FALSE" in ``config.sh``. 
```

Comment:
Change to "Users who have already staged the **observation data needed for METplus**..."